modules/console/pages/flow-console.adoc

```diff
@@ -0,0 +1,114 @@
+
+* Is there a site with more or cheaper resources that you want to use?
+For example, if you have two sites, A and B, and resources are more expensive on site A, you might want to locate the flow collector and console on site B.
+--
+
+. Create a site with the flow collector and console enabled:
++
+----
+$ skupper init --enable-console --enable-flow-collector
+----
++
+NOTE: You must enable the console on the same site as you enable the flow collector.
+
+
+// Type: procedure
+[id="accessing-console"] 
+== Accessing the Skupper console
+
+By default, the Skupper console is protected by credentials available in the `skupper-console-users` secret.
+
+
+.Procedure
+
+. Determine the Skupper console URL using the `skupper` CLI, for example:
++
+----
+$ skupper status
+Skupper is enabled for namespace "west" in interior mode. It is not connected to any other sites. It has no exposed services.
+The site console url is:  https://skupper-west.apps-crc.testing
+----
+
+. Browse to the Skupper console URL. 
+The credential prompt depends on how the site was created using `skupper init`:
++
+* using the `--console-auth unsecured` option, you are not prompted for credentials.
+* using the `--console-auth openshift` option, you are prompted to enter OpenShift cluster credentials.
+* using the default or `--console-user <user>  --console-password <password>` options, you are prompted to enter those credentials.
+
+. If you created the site using default settings, that is `skupper init`, a random password is generated for the `admin` user.
+To retrieve the password the `admin` user:
++
+----
+$ kubectl get secret skupper-console-users -o jsonpath={.data.admin} | base64 -d
+JNZWzMHtyg
+----
+
+
+// Type: procedure
+[id="exploring-console"] 
+== Exploring the Skupper console
+
+The Skupper console provides an overview of the following:
+
+* Sites - {skupper-name} installations on the current {service-network}.
+* Components - services that are exposed on the {service-network}
+* Processes - running processes relating to exposed services.
+* Addresses - services and port numbers relating to exposed services
+* Network - visualizations of sites, components and processes
+
+The Skupper console also provides useful networking information about the {service-network}, for example, traffic levels between sites.
+
+image::{img-loc}skupper-adservice.png[]
+
+
+. Check the *Sites* tab, all your sites should be listed.
+
+. Check that all the services you exposed are visible in the *Components* tab.
+
+. Click a component to show the component details and associated processes.
+
+. Click on a process to display the process traffic.
+NOTE: The process detail displays the associated image, host and addresses.
+
+. Click an address to show the details for that address.
+
+To view a static representation of all the console features, see the https://skupper-console-vry5.vercel.app/[Example Console].
```

Comment:
I think we can skip/delete this info.
This link is automatically updated when the main branch is updated. We risk to see a 'beta' console version compared to the versione released.

Reply:
I agree, commenting out, so it's still available to users who view docs source